docker-compose.dev.yml

```diff
@@ -24,6 +24,12 @@ services:
     depends_on:
       postgres:
         condition: service_started
+  theme_classifier:
+    image: "openstatefoundation/ori-theme-classifier:v1.0.0"
+    ports:
+      - "80:5000"
+    environment:
+      - "CLASSIFIER_HOST=localhost"
```

Comment:
It should be added to the `backend` container instead ;)

Reply:
Oops, you're right.